README.md

```diff
@@ -26,9 +26,11 @@ After following the steps outlined below, your fellow devs will be able to clone
 
 1. Download and copy the files from this repo (except for this README) into the project folder — do not clone the repo.
 2. In the project root, run `git submodule add git@github.com:fostermadeco/ansible-roles.git ansible/roles`
-3. Set your project variables in `ansible/group_vars/all`.
+3. Set your project variables in `ansible/group_vars/all`
 4. When `vagrant up` provisions the development machine to your satisfaction, make the initial commit.
 
+**Note: When configuring a new project, verify that MySQL and PHP versions in your configuration file match the MySQL and PHP versions on the server that the project will be deployed to. Please review the Available Config section below for information about how to modify these versions.**
```

Comment:
@alexmglover Should this say software versions including but not limited to MySQL and PHP rather than just specifying MySQL and PHP? I worry that we'll run into conflicts with other software as well.

Reply:
@smaida Are you referring to apt package versions?

Reply:
Yes. Are there cases other than MySQL and PHP where the software versions matter? Maybe that is not a concern.